### PR TITLE
Fix Clone Action of Configuration is out of place

### LINKED ISSUE #13874: Sidecar: Clone Action of Configuration is out of place
## Expected Behavior
The clone action of the drop down field of a Sidecar configuration should be aligned like delete.

## Current Behavior
It is a bit to the left.

Bildschirmfoto vom 2022-11-03 13-50-55

## Possible Solution
Needs some CSS :heart: 

## Your Environment
* Graylog Version: Graylog 5.0.0-beta.2+d576a3d (Oracle Corporation 17.0.2 on Linux 5.15.0-52-generic)

### PR DESCRIPTION
**Pull request by gally47**
## Motivation and Context

fixes #13874

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by ousmaneo — CHANGES_REQUESTED** [review]
Looks good, I think we can take the chance to move the file to `Typescript`.

File: changelog/unreleased/issue-13874.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type = "fixed"
+message = "Fix Clone Action of Configuration is out of place"
```
Comment (ousmaneo) [inline changelog/unreleased/issue-13874.toml:2]:
```suggestion
message = "Fix Clone Action style on Collectors configurations list "
```

**Review by mnin — APPROVED** [review]
LGTM

**Review by mnin — APPROVED** [review]
LGTM

please use `git merge` instead of `git rebase` after first review was started, to give the reviewer to track the newest changes

**Comment by gally47** [comment]
> Looks good, I think we can take the chance to move the file to `Typescript`.

@ousmaneo is on vacation but he originally requested changes. @mnin reviewed the requested changes and approved them. So, I'll (try to) dismiss @ousmaneo 's review as I addressed the comments, so I can merge this PR. Leaving this comment here to clarify why I dismiss a review.

**Review by dennisoelkers — APPROVED** [review]

**Pull request merged**